ownCloud/Gallery/GalleryHostViewController.swift

```diff
@@ -0,0 +1,229 @@
+//
+//  AlternativePageViewController.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 14/02/2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import ownCloudSDK
+
+class GalleryHostViewController: UIPageViewController {
+
+	typealias Filter = ([OCItem]) -> [OCItem]
+
+	// MARK: - Constants
+	let hasChangesAvailableKeyPath: String = "hasChangesAvailable"
+	let imageFilterRegexp: String = "\\A((image/(?!(gif|svg*))))"
+
+	// MARK: - Instance Variables
+	weak private var core: OCCore?
+	private var selectedItem: OCItem
+	private var items: [OCItem]?
+	private var query: OCQuery
+	private weak var viewControllerToTansition: DisplayViewController?
+	private var selectedFilter: Filter?
+
+	// MARK: - Filters
+	lazy var filterImageFiles: Filter = { items in
+		let filteredItems = items.filter({$0.type != .collection && $0.mimeType?.matches(regExp: self.imageFilterRegexp) ?? false})
+		return filteredItems
+	}
+
+	lazy var filterOneItem: Filter = { items in
+		let filteredItems = items.filter({$0.type != .collection && $0.fileID == self.selectedItem.fileID})
+		return filteredItems
+	}
+
+	// MARK: - Init & deinit
+	init(core: OCCore, selectedItem: OCItem, query: OCQuery) {
+		self.core = core
+		self.selectedItem = selectedItem
+		self.query = query
+
+		super.init(transitionStyle: .scroll, navigationOrientation: .horizontal, options: nil)
+		query.addObserver(self, forKeyPath: hasChangesAvailableKeyPath, options: [.initial, .new], context: nil)
+		Theme.shared.register(client: self)
+	}
+
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	deinit {
+		query.removeObserver(self, forKeyPath: hasChangesAvailableKeyPath)
+		Theme.shared.unregister(client: self)
+	}
+
+	// MARK: - ViewController lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+		dataSource = self
```

Comment:
This is my Xcode indentation config since the beginning of the project 
![screenshot 2019-03-06 at 08 49 38](https://user-images.githubusercontent.com/8929738/53864455-cf0bce80-3fec-11e9-9c1c-44e2b5f83667.png)

Reply:
@pablocarmu at least this is what i see in github, but may be it is rather something related to github code viewer settings.